src/looklike.ml

```diff
@@ -0,0 +1,150 @@
+(** Looklike allows you to find recurrent patterns in some lists *)
+
+
+(** Module type which allows comparing elements *)
+module type EqType = 
+sig
+  type t
+  val equal : t -> t -> bool
+  val nil : t 
+  val get_str : t -> string
+end
+
+
+
+(** OCaml type to get pattern and its position *)
+type 'a index_string = {
+  depot : 'a list;
+  pos_doc :int ;
```

Comment:
Please put a space after a colon.

Reply:
Can you comment the code to explain the role of each field? I am not sure to infer their meaning from their identifiers alone.